s/babylon/theater/utils/make-babylon-world.ts

```diff
@@ -46,11 +46,11 @@ export function makeBabylonWorld(): BabylonWorld {
 		renderLoop,
 		sceneInstrumentation,
 		engineInstrumentation,
-		resize() {
+		resize(res) {
```

Comment:
something minor, but maybe this could be fully named? `resolution`

Reply:
Yeah i think it would be better to use full name here, im starting to realize it more, its only better to use shortands only for obvious things i think ..

Reply:
yeah
the classic is `e` for `event`, when writing event listeners 😅